heareval/predictions/runner.py

```diff
@@ -23,6 +23,13 @@
     nargs=-1,
     required=True,
 )
+@click.option(
+    "--outdir",
+    default=None,
+    help="Output directory to save prediction results. (Defaults to "
```

Comment:
Seems like it will (and SHOULD) be a separate directory from input embeddings:

```
predictions/embedding_name/task_name
```

Reply:
This was an old commented -- updated